hexagonal/src/main/java/com/iluwatar/hexagonal/database/MongoTicketRepository.java

```diff
@@ -169,10 +169,9 @@ public MongoDatabase getMongoDatabase() {
   public Map<LotteryTicketId, LotteryTicket> findAll() {
     Map<LotteryTicketId, LotteryTicket> map = new HashMap<>();
     List<Document> docs = ticketsCollection.find(new Document()).into(new ArrayList<Document>());
-    for (Document doc: docs) {
-      LotteryTicket lotteryTicket = docToTicket(doc);
+    docs.stream().map(lotteryTicket -> docToTicket(doc)).forEach(lotteryTicket -> {
```

Comment:
Can you break it into multiple lines for easy readability? Like:
docs.stream()
    .map(...)
    .forEach(...)